docs/user/AnsibleCloudAuthentication.md

```diff
@@ -0,0 +1,47 @@
+# Cloud Authentication
+
+The docker container contains cloud clis needed for interacting with the various clouds.
```

Comment:
"The docker container includes cloud provider clis needed for interacting with the various cloud providers."

Reply:
updated